resources/js/Hooks/useMetaMask.ts

```diff
@@ -62,7 +62,8 @@ const networkConfigs: Record<number, object> = {
     },
 };
 
-const hasMetaMask = (): boolean => window.ethereum !== undefined;
+// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
+const hasMetaMask = (): boolean => window.ethereum?.isMetaMask === true;
```

Comment:
```suggestion
const hasMetaMask = (): boolean => window.ethereum?.isMetaMask ?? false
```
try this? then it defaults to a boolean still in case window.ethereum is not defined 🤔

Reply:
causes other lint issues, reverting